### PR TITLE
map fits to boundary of features added

### PR DESCRIPTION
**Pull request by desire-geogecko**
Loops through all features of a farm and zooms map to fit all features within view

### CODE REVIEW

File: js/farms.js [hunk 1]
```diff
@@ -63,21 +63,47 @@ function handleWebsiteJson_(data) {
 
   set_mgt_zones.clear()
   mgt_zones.length = 0
+
+  var bounds = [];
```
Comment (RonKbS) [inline js/farms.js]: Can we stick with 'let' as much as possible rather than var. For this and line 85,...my liege

File: js/farms.js [hunk 2]
```diff
@@ -63,21 +63,47 @@ function handleWebsiteJson_(data) {
...
+  bounds.forEach(function(d, i) {
+
+      // find corners
+      if (minlat > d._southWest.lat) minlat = d._southWest.lat;
+      if (minlon > d._southWest.lng) minlon = d._southWest.lng;
+      if (maxlat < d._northEast.lat) maxlat = d._northEast.lat;
+      if (maxlon < d._northEast.lng) maxlon = d._northEast.lng;
+  });
+  
+  c1 = L.latLng(minlat, minlon);
```
Comment (RonKbS) [inline js/farms.js]: These guys seem to be missing titles 🌚

**Review by RonKbS — COMMENTED** [review]
Subarashi, desu!!!

Couple of things;
Can we also have the zoom out restricted. Pete had an issue where he would try to scroll down but instead zoom out.

`map-bounds` might also not have been based off of develop. The last card is missing and on checking in the console. It's trying to pull data from an invalid source that I changed. Updating `develop` and then rebasing `map bounds` should fix this.

File: js/farms.js (hunk 2 above)
Comment (Geo-Gecko) [reply js/farms.js]: let there be let

File: js/farms.js (hunk 1 above)
Comment (Geo-Gecko) [reply js/farms.js]: let there be let

**Review by RonKbS — APPROVED** [review]

**Pull request merged**